### ISSUE TITLE
connection refused when attempting to launch tool

### ISSUE THREAD

**Issue by Ninjastorm380** [rcpoison/dxvk-cache-pool#23]
attempted to run your tool, but only got an error. help?

"target directory is: /home/demon4hire/.cache/dxvk-cache-pool
java.net.ConnectException: Connection refused"

**Comment by dannydogg** [comment]
I need some help with the exact same issue, too :)

**Comment by VeryOriginalNonsense** [comment]
Server is down again today (it seems). I don't know of alternative servers, so I'm kind of sitting on my hands here.
Never used the tool & am hoping it hasn't been down since last year…

Any alternatives?
Tooling or service?

**Comment by drewdemarest** [comment]
I figured I'd give this tool a shot as well and it looks like the server is still down.

**Comment by VeryOriginalNonsense** [comment]
For now what I found to be useful is [this repo](https://github.com/begin-theadventure/dxvk-caches).
I found it by looking for a repo in r/linux_gaming.
I also noticed that some Lutris scripts pull a file from some random bucket somewhere. This seems a bit more sketchy to me.

I'm new to Linux gaming so I didn't know these things. Hopefully this is actually helpful and not common knowledge.

**Comment by drewdemarest** [comment]
Okay so I did some tinkering with dxvk cache pool and noted that we can compile both a client AND a server binary.
It looks like this tool is intended as a method by which users can upload their dxvk caches to a central repository server.
Since dxvk only compiles shaders only for what it renders, it makes sense to merge caches from a bunch of people as it builds a more complete cache that anyone can download.
All the caches get merged server side and then clients can download from the merged 'pool' for a given game. Pretty neat stuff.

It also explains all the shader updates that steam downloads for my proton games. I imagine people are constantly uploading their dxvk caches to Valve.

Furthermore, it's not compatible with the latest dxvk v 1.10.1 that I'm running w/o some changes to the code, I think it only goes up to 1.2.x.

tldr; without a fork and community effort this tool won't work. Not really useful in the client side capacity. Only work with dxvk 1.2.x
